packages/engine/Specs/Renderer/TextureAtlasSpec.js

```diff
@@ -832,16 +832,16 @@ describe("Scene/TextureAtlas", function () {
         expect(drawAtlas(atlas, [index0, index1, index2, index3])).toBe(
           `
 ••••••••••••••••••••••••••••••••
-•2222222222222222333333333301•••
-•22222222222222223333333333•••••
-•22222222222222223333333333•••••
-•22222222222222223333333333•••••
-•22222222222222223333333333•••••
-•22222222222222223333333333•••••
-•22222222222222223333333333•••••
-•22222222222222223333333333•••••
-•22222222222222223333333333•••••
-•22222222222222223333333333•••••
+•2222222222222222•3333333333•0••
+•2222222222222222•3333333333••••
+•2222222222222222•3333333333•1••
```

Comment:
When I reviewed #13037 I assumed this representation is "top down" as it would be rendered. However if I understand the change here (which I might not) this seems to be adding an extra row of pixels to the "right" in this representation but the "bottom" in the actual billboard. Do I just not understand how the texture atlas works? Or should this representation be rotated?

Reply:
Ah, the origin is bottom-left and I'd assumed top-left. I don't think that changes anything implementation-wise. But certainly debugging will be easier if the test output uses the right representation, so I'll update the test output to match.

Will respond to the other part of this in the main comment thread.

Reply:
Both PRs have been updated so that the `drawAtlas` test output matches the coordinate system in TextureAtlas.js, origin = bottom-left. Thanks @jjspace!